wade/chain.py

```diff
@@ -105,17 +108,21 @@ def state_sanitize(self):
         """
 
         try:
-            self._func = self._store.get_op(self._cmd.op)
-            if not self._func:
-                raise RespondWithError(
-                    'INTERNAL',
-                    "undefined op: %s" % self._cmd.op,
-                    {
-                        'id': self._my_id,
-                        'op': self._cmd.op,
-                    },
-                    "Client and node versions possibly differ.",
-                )
+            if self._cmd.op == 'FULL_SYNC':
+                self._op_type = SYNC_OP
+            else:
+                self._func = self._store.get_op(self._cmd.op)
+                self._op_type = self._func._op_type
```

Comment:
if we're not really using the func._op_type here maybe we should re-envision how the func is defined / what properties it has.